PWGHF/D2H/TableProducer/candidateCreatorCharmResoReduced.cxx

```diff
@@ -332,27 +348,121 @@ struct HfCandidateCreatorCharmResoReduced {
           rowCandidateResoTrack(pVecD[0], pVecD[1], pVecD[2],
                                 candV0Tr.px(), candV0Tr.py(), candV0Tr.pz(),
                                 invMassReso,
-                                invMassD);
+                                invMassD - invMassD0);
```

Comment:
it seems to me from line 182 and 184 that `invMassD` already is the delta mass (so here we are subtracting the D0 mass twice). Personally I would keep the `invMassD` without the D0 mass subtraction and I would define another variable `deltaInvMass` to be used when needed for clarity.